pages/understanding-json-schema/structuring.md

```diff
@@ -565,3 +565,8 @@ In Draft 4-7, a subschema `$id` is just a base URI change and not
    only allowed at the root of a Schema Resource, all schemas bundled
    using subschema `$id` must use the same dialect.
 </Infobox>
+
+<Infobox label="Draft-specific info">
+In Draft 2020-12,`$id` support for changing dialects in an embedded schema 
+   (using $schema with a different value than the parent schema) was added.   
```

Comment:
The way `$id` is used here confuses me. I think it makes more sense just to drop it.

```suggestion
In Draft 2020-12, support for changing dialects in an embedded schema 
   (using `$schema` with a different value than the parent schema) was added.   
```